esque/cli/options.py

```diff
@@ -34,10 +37,37 @@ def cluster(self):
             self._cluster = Cluster()
         return self._cluster
 
+    def _get_verbose(self) -> bool:
+        if "ESQUE_VERBOSE" in os.environ:
```

Comment:
We have https://github.com/real-digital/esque/blob/master/esque/cli/environment.py for these. :)

Reply:
Yeah... this is just impossible to mock if you have `from esque.cli.environment import ENV_VAR` all over the place. You'll have to find all imports and mock them. Unless you somehow manage to enforce that its just `from esque.cli import environment` everywhere and we only access the vars via the module.
That's why I wanted to avoid it. But I see why we should keep it consistent... I'll change it to use that module and put a comment about mocking the variables there.